client/src/main/js/controller/dialog/ImagemController.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    'use strict';
+
+    angular.module('localModulo').controller('ImagemController', ['imagem', 'editavel', '$rootScope', '$mdDialog', function (imagem, editavel, $rootScope, $mdDialog) {
```

Comment:
aí tu recebe aqui a função de callback e só executa ela no excluir imagem, dependendo de como for ficar os parâmetros.

ou adaptar para outra solução